packages/drawer/README.md

```diff
@@ -112,6 +115,164 @@ class MyApp extends Component {
 }
 ```
 
+##### Usage with Top App Bar
```

Comment:
Since this is general to permanent or dismissible drawer, make this a level 4 heading and remove all references to "dismissible" below